test/one/profiler/test/SourceLocator.java

```diff
@@ -0,0 +1,68 @@
+package one.profiler.test;
```

Comment:
We badly need a linter for file headers.

Reply:
Can we merge https://github.com/async-profiler/async-profiler/pull/1016 if I remove the trailing whitespace checker for now(it is scanning .class files and giving false results)